product_docs/docs/pgd/5/consistency/column-level-conflicts/03_timestamps.mdx

```diff
@@ -7,47 +7,46 @@ As previously mentioned, column-level conflict resolution depends on a timestamp
-
-As the current timestamp is unrelated to the commit timestamp, using it to resolve conflicts means that the result isn't equivalent to the commit order, which means it is probably cannot be serialized.
+<!--second sentence needs clarification>
+As the current timestamp is unrelated to the commit timestamp, using it to resolve conflicts means that the result isn't equivalent to the commit order, which means it probably can't be serialized.
 
 When using current timestamps to order changes or commits, the conflicting changes might have exactly the same timestamp because two or more nodes happened to generate the same timestamp. This risk isn't unique to column-level conflict resolution, as it can happen even for regular row-level conflict resolution. The node id is used as the tiebreaker in this situation. The higher node id wins. This approach ensures that the same changes are applied on all nodes.
 
 ### `column_commit_timestamp`
 
-You can also use the actual commit timestamp, specified with `column_commit_timestamp` as the conflict detection method. This has the advantage of using the commit time, which will be the same for all changes made within an `UPDATE`.
+You can also use the actual commit timestamp specified with `column_commit_timestamp` as the conflict detection method. This approach has the advantage of using the commit time, which is the same for all changes made in an `UPDATE`.
 
 !!! Note
 Statement transactions might be added in the future, which will address issues with mixing effects of concurrent statements or transactions. Still, neither of these options can ever produce results equivalent to commit order.
 !!!
 
 ## Inspecting column timestamps
 
-The column storing timestamps for modified columns is maintained by triggers. Don't modify it directly. It can be useful to inspect the current timestamps value, for example, while investigating how a conflict was resolved.
+The column storing timestamps for modified columns is maintained by triggers. Don't modify it directly. It can be useful to inspect the current timestamp's value, for example, while investigating how a conflict was resolved.
 
 !!! Note 
-The timestamp mapping is maintained by triggers and the order in which triggers execute matters. So if you have custom triggers that modify tuples and are executed after the pgl_clcd_ triggers, the modified columns aren't detected correctly.
+The timestamp mapping is maintained by triggers, and the order in which triggers execute matters. So if you have custom triggers that modify tuples and are executed after the pgl_clcd_ triggers, the modified columns aren't detected correctly.
```

Comment:
```suggestion
The timestamp mapping is maintained by triggers, and the order in which triggers execute matters. If your custom triggers modify tuples and are executed after the pgl_clcd_ triggers, the modified columns aren't detected correctly.
```

Reply:
Can you add a conclusion sentence to this note to indicate what the best practice is to avoid this?